chapters/inheritance.tex

```diff
@@ -690,6 +690,8 @@ \section{Redeclaration}\label{redeclaration}
 the right of the component; and thus can be redeclared in a consistent
 way.
 
+Having annotations on the \lstinline!redeclare! construct in a modifier is deprecated and the annotations have no effect.
```

Comment:
This sounds a little bit like there used to be an affect which is not even mentioned any longer.  Maybe this can be avoided by writing it this way?
```suggestion
Presence of annotations on the \lstinline!redeclare! construct in a modifier is deprecated, but this only affects vendor-specific annotations as none of the annotations in the specification ever had a meaning in this context.
```

Reply:
I will reformulate it a bit.

Reply:
OK.  Resolving.